shogun-lib/src/main/java/de/terrestris/shogun/lib/service/security/provider/keycloak/KeycloakUserProviderService.java

```diff
@@ -47,7 +47,7 @@
  */
 @ConditionalOnExpression("${keycloak.enabled:true}")
 @Log4j2
-@Component
+@Component("keycloakUserProviderService")
```

Comment:
Is this really needed (since it should be the default)?

Reply:
is not needed, removed in fixed in 63741cd4c7a03a279856fdbdf9e73df0cd855a95